test-app/frontend/src/ManageVolunteers/ManageVolunteers.tsx

```diff
@@ -0,0 +1,45 @@
+import VolunteerCard from './VolunteerCard';
+
+export interface Volunteer
+{
+  /*
+    Mock schema for a volunteer
+    to be displayed when rendering all
+    the volunteers.
+    */
+  firstName: string;
+  lastName: string;
+  userID: string;
+  eventsAttending: string[];
+
+}
+
+interface ManageVolunteerProps
+{
+  volunteers: Volunteer[]
+}
+
+function renderCards(volCardLst: Volunteer[]) {
+  // This function will loop through the ManageVolProps list
+  // and render each card
+  const jsxList = [];
+  for (let i = 0; i < volCardLst.length; i++) {
+    jsxList.push(
+      <VolunteerCard {...volCardLst[i]}/>,
+    );
+  }
+  return (
+    jsxList
+  );
+}
+
+export default function ManageVolunteers(props: ManageVolunteerProps) {
+  return (
+    <div>
+      <h1>Volunteer List</h1>
+      <ul>
+        {renderCards(props.volunteers)}
```

Comment:
This is not typical for rendering components. If you want to make a new component for rendering each Volunteer, you should make that a component instead. Given what `renderCards` is, I'd suggest skipping that step since `VolunteerCard` already exists and just do what `renderCards` does inline.

```
<ul>
  {props.volunteers.map((volunteer) => <VolunteerCard {...volunteer} />)}
</ul>
```